crmsh/ui_cluster.py

```diff
@@ -428,6 +428,8 @@ def do_join(self, context, *args):
         options, args = parse_options(parser, args)
         if options is None or args is None:
             return
+        if any([utils.service_is_active(service) for service in ["sbd", "corosync", "pacemaker", "corosync-qdevice"]]):
+            return
```

Comment:
For the better user experience, before returning, to print a message up front is useful for the system admin to take the proper action(s) to react on it. Something like,

```suggestion
            logger.warning(" '{}' service must stop to proceed.".format(service)
            return False
```

Reply:
Thanks!